mmhuman3d/utils/mesh_utils.py

```diff
@@ -117,40 +186,50 @@ def save_meshes_as_plys(meshes: Meshes = None,
     else:
         if verts is not None or faces is not None or verts_rgb is not None:
             warnings.warn('Redundant input, will use meshes only.')
-
-    assert len(paths) >= len(meshes), 'Not enough output paths.'
+    assert files is not None
+    if not isinstance(files, list):
+        files = [files]
+    assert len(files) >= len(meshes), 'Not enough output files.'
     writer = IO()
-    if not isinstance(paths, list):
-        paths = [paths]
     for idx in range(len(meshes)):
-        assert paths[idx].endswith('.ply'), 'Please save as .ply files.'
+        assert files[idx].endswith('.ply'), 'Please save as .ply files.'
         writer.save_mesh(
-            meshes[idx], paths[idx], colors_as_uint8=True, binary=False)
+            meshes[idx], files[idx], colors_as_uint8=True, binary=False)
 
 
-def save_meshes_as_objs(meshes: Meshes = None, paths: List[str] = []) -> None:
-    """Save meshes as .obj files. Mainly for uv texture meshes.
+def save_meshes_as_objs(meshes: Meshes = None,
+                        files: List[str] = None) -> None:
```

Comment:
Same as above. Files should not be None.

Reply:
ok, but `[]` should not be used as a default value, maybe use `()`